include/deal.II/distributed/grid_refinement.h

```diff
@@ -152,8 +152,8 @@ namespace parallel
       template <int dim, typename Number, int spacedim>
       void
       refine_and_coarsen_fixed_number(
-        parallel::distributed::Triangulation<dim, spacedim> &tria,
-        const dealii::Vector<Number> &                       criteria,
+        Triangulation<dim, spacedim> & tria,
+        const dealii::Vector<Number> & criteria,
```

Comment:
The initial comment of this function still says `Like dealii::GridRefinement::refine_and_coarsen_fixed_number, but for parallel distributed triangulations.` - I am wondering if we can use a simpler-to-recognize name, e.g. `Like dealii::GridRefinement::refine_and_coarsen_fixed_number, but designed for parallel computations.` Together with the rest of the description, this would seem to be more accurate.